docs/components/movement-sensor/adxl345.md

```diff
@@ -100,37 +143,39 @@ Edit and fill in the attributes as applicable.
+| `interrupt_pin`     | string | **Required** | The `name` of the [digital interrupt](/components/board/#digital_interrupts) you configured for the pin on the [board](/components/board/) wired to the `accelerometer_pin`. |
+| `exclude_x`         | bool   | Optional     | Tap detection defaults to all three axes. Exclude the x axis by setting this to true. <br> Default: `false` |
+| `exclude_y`         | bool   | Optional     | Tap detection defaults to all three axes. Exclude the y axis by setting this to true. <br> Default: `false` |
+| `exclude_z`         | bool   | Optional     | Tap detection defaults to all three axes. Exclude the z axis by setting this to true. <br> Default: `false` |
+| `threshold`         | float  | Optional     | The magnitude of the threshold value for tap interrupt (in milligrams, between 0 and 15,937). <br> Default: `3000` |
 | `dur_us`            | float  | Optional     | Unsigned time value representing maximum time that an event must be above the `threshold` to qualify as a tap event (in microseconds, between 0 and 159,375). <br> Default: `10000` |
 
 ### Freefall attributes
 
 Inside the `freefall` object, you can include the following attributes:
 
-| Name                | Type   | Inclusion    | Default Value                                                                                                                                                                                            | Description |
-| ------------------- | ------ | ------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------- |
-| `accelerometer_pin` | int    | **Required** | On the accelerometer you can choose to send the interrupts to int1 or int2. Specify this by setting this config value to `1` or `2`.                                                                     |
-| `interrupt_pin`     | string | **Required** | The `name` of the [digital interrupt](/components/board/#digital_interrupts) you configured for the pin on the [board](/components/board/) wired to the `accelerometer_pin`.                             |
-| `threshold`         | float  | Optional     | The acceleration on each axis is compared with this value to determine if a free-fall event occurred (in milligrams, between 0 and 15,937). <br> Default: `437.5`                                        |
+<!-- prettier-ignore -->
+| Name                | Type   | Inclusion    | Default Value | Description |
+| ------------------- | ------ | ------------ | ------------- | ----------- |
+| `accelerometer_pin` | int    | **Required** | On the accelerometer you can choose to send the interrupts to int1 or int2. Specify this by setting this config value to `1` or `2`. |
```

Comment:
```suggestion
| `accelerometer_pin` | int    | **Required** | On the accelerometer, you can choose to send the interrupts to `int1` or `int2`. Specify this by setting this config value to `1` or `2`. |
```

Reply:
As below, I think it's nice for the monospace to only highlight the values that one needs to type in. int1 and int2 are labels on the breakout board, not actual code or JSON values. Do you think they should be in quote marks or something?